src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -26,14 +26,15 @@ spec:
         port: 8080
 
       # Temp workaround for any cluster pod
-      # @todo: remove this once cluster pods is a remote generated target
+      # todo: remove this once cluster pods is a remote generated target
       - description: "Keycloak backchannel access"
         direction: Ingress
         selector:
           app.kubernetes.io/name: keycloak
         remoteGenerated: Anywhere
         port: 8080
 
```

Comment:
Not commenting on the right lines here, but might suggest we lockdown the "PostgreSQL Database access" policy with conditionals for internal, cidr, or anywhere (similar to storage conditionals for velero/loki).